pandora-client-web/src/components/settings/securitySettings.tsx

```diff
@@ -0,0 +1,273 @@
+import React, { ReactElement } from 'react';
+import { AssertNever, FormatTimeInterval, PasswordSchema, IDirectoryAccountInfo, IClientDirectoryNormalResult } from 'pandora-common';
+import { useForm } from 'react-hook-form';
+import { useCurrentAccount, useDirectoryConnector } from '../gameContext/directoryConnectorContextProvider';
+import { PrehashPassword } from '../../crypto/helpers';
+import { Form, FormCreateStringValidator, FormField, FormFieldError } from '../common/form/form';
+import { Button } from '../common/button/button';
+import { FieldsetToggle } from '../common/fieldsetToggle/fieldsetToggle';
+import { useAsyncEvent } from '../../common/useEvent';
+import { ModalDialog, useConfirmDialog } from '../dialog/dialog';
+import { useObservable } from '../../observable';
+import type { AuthToken } from '../../networking/directoryConnector';
+import { useCurrentTime } from '../../common/useCurrentTime';
+import { useKeyDownEvent } from '../../common/useKeyDownEvent';
+import { Row } from '../common/container/container';
+
+export function SecuritySettings(): ReactElement | null {
+	const account = useCurrentAccount();
+
+	if (!account)
+		return <>Not logged in</>;
+
+	return (
+		<>
+			<ConnectedClients />
+			<PasswordChange account={ account } />
+		</>
+	);
+}
+
+type AccountConnectedClients = IClientDirectoryNormalResult['queryConnections']['connections'];
+
+function ConnectedClients(): ReactElement {
```

Comment:
A message to logout all clients was added to the message definitions, but no button was added for it.
It would be nice to have it added - maybe on the same row as "refresh", just on the right?